services/proxy/pkg/middleware/authentication.go

```diff
@@ -147,7 +147,9 @@ func configureSupportedChallenges(options Options) {
 func writeSupportedAuthenticateHeader(w http.ResponseWriter, r *http.Request) {
 	caser := cases.Title(language.Und)
 	for _, s := range SupportedAuthStrategies {
-		w.Header().Add(WwwAuthenticate, fmt.Sprintf("%v realm=\"%s\", charset=\"UTF-8\"", caser.String(s), r.Host))
+		if !(s == "basic" && r.Header.Get("XmlHttpRequest") != "") {
```

Comment:
Something is odd with this condition. I still see the `Basic` and the `Bearer` auth header when running the curl from the ticket.

Reply:
fixed it.